R/helper-functions.R

```diff
@@ -184,7 +184,7 @@ checkRoot <- function(cluster_df) {
   # Handle Forests
   cols <- colnames(cluster_df)
   if (length(unique(cluster_df[[1]])) > 1) {
-    cluster_df$root <- "AllClusters"
+    cluster_df$root <- "ClusterAll"
```

Comment:
why change the root node name ? AllClusters makes more sense to me than ClusterAll

Reply:
in the Plot function, we remove the `cluster` prefix and show only the suffix of node name. For example, if a node is `Cluster1C7`, this will only show `1C7` in the plot. So I renamed it to make it consistent for plotting.

Reply:
thats fine, root node can be called `AllClusters`

Reply:
Then in the plot it'll show `tesrs` as the root node

Reply:
I see what you mean, then `ClusterAllClusters`  😄